private_sharing/serializers.py

```diff
@@ -116,3 +116,23 @@ def to_representation(self, obj):
             rep.pop("username")
 
         return rep
+
+
+class DataTypeSerializer(serializers.ModelSerializer):
+    """
+    Serialize data for a project member.
+    """
+
+    class Meta:  # noqa: D101
+        model = DataType
+
+        fields = ["id", "name", "parent", "description", "projects"]
+
+    projects = serializers.SerializerMethodField()
+
+    def get_projects(self, obj):
```

Comment:
This needs to distinguish between source_projects and requesting_projects. (It's just that so far only source has been implemented, but the intent is to include requesting ones as well.)

Reply:
Roger that